### PR TITLE
Add celery and backups on ansible

### PR DESCRIPTION
**Pull request by sirodoht**
This PR adds the following on our ansible playbook:

* redis server with default config
* celery running with django as a systemd service
* daily backups using pg_dump and cron

### CODE REVIEW

File: ansible/playbook.yaml [hunk 1]
```diff
@@ -151,24 +167,39 @@
...
-    - name: systemd enable
+    - name: web systemd enable
       ansible.builtin.systemd:
         name: sfs
         enabled: yes
-    - name: systemd start
+    - name: web systemd start
       ansible.builtin.systemd:
         name: sfs
         state: restarted
```
Comment (nicksellen) [inline ansible/playbook.yaml:194]: I know this is off topic for the PR, but just mentioning it in passing...

One day can add ansible handlers to only restart when something has changed that impacts it https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_handlers.html

An example from karrot:
- this task calls "notify" after changing the env config https://github.com/karrot-dev/yuca/blob/26e36067c42290481a654bf42dc79646c3e249cc/roles/karrot-backend/tasks/main.yml#L372
- it causes this handler to run and restart https://github.com/karrot-dev/yuca/blob/26e36067c42290481a654bf42dc79646c3e249cc/roles/karrot-backend/handlers/main.yml#L3-L8

File: ansible/sfsworker.service.j2 [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Environment="CELERY_BROKER_URL={{ celery_broker_url }}"
+Environment="CELERY_RESULT_BACKEND={{ celery_result_backend }}"
+Environment="ENABLE_ALLAUTH_SOCIAL_LOGIN={{ enable_social_login }}"
+Environment="GOOGLE_CLIENT_ID={{ google_client_id }}"
+Environment="GOOGLE_SECRET={{ google_secret }}"
+Environment="EMAIL_HOST={{ email_host }}"
+Environment="EMAIL_HOST_USER={{ email_host_user }}"
+Environment="EMAIL_HOST_PASSWORD={{ email_host_password }}"
+Environment="DEFAULT_FROM_EMAIL={{ default_from_email }}"
+Environment="WEATHER_API_KEY={{ weather_api_key }}"
```
Comment (nicksellen) [inline ansible/sfsworker.service.j2:28]: We already have an environment file, no? If so, then can use systemd's `EnvironmentFile=` option https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html#EnvironmentFile=

#NiceToHave

Advantage: one definition file that specifies the environment

File: ansible/playbook.yaml [hunk 3]
```diff
@@ -112,12 +113,27 @@
...
+        path: /home/deploy/backups
+        state: directory
+        mode: '0755'
+      become_user: deploy
+    - name: Add job to crontab
+      ansible.builtin.cron:
+        name: backup postgres dump
+        minute: "00"
+        hour: "5"
+        job: "PGPASSWORD={{ postgres_password }} /var/www/shared-futures-space/backup-database.sh"
```
Comment (nicksellen) [inline ansible/playbook.yaml:128]: I notice there's no backup for user uploaded files included (yet?).

**Review by nicksellen — APPROVED** [review]
Approved! All suggestions optional ☺️

File: ansible/playbook.yaml (hunk 1 above)
Comment (sirodoht) [reply ansible/playbook.yaml:194]: I see, excellent, taking note!

File: ansible/sfsworker.service.j2 (hunk 2 above)
Comment (sirodoht) [reply ansible/sfsworker.service.j2:28]: Nice one, yes, I think we can improve with this! Thanks!

File: ansible/playbook.yaml (hunk 3 above)
Comment (sirodoht) [reply ansible/playbook.yaml:128]: Hmm yeah good point, which reminds me we also need to rework the avatar images, which are currently user uploaded files.

**Pull request merged**